libs/sdk-core/src/models.rs

```diff
@@ -1213,10 +1214,18 @@ pub enum SwapStatus {
     /// eligible to be redeemed normally.
     Initial = 0,
 
+    WaitingConfirmation = 1,
+
+    Redeemable = 2,
+
+    Redeemed = 3,
+
     /// The swap address has confirmed transactions associated with it and the lock timeout has passed since
     /// the earliest confirmed transaction. This means the only way to spend the funds from this address is by
     /// broadcasting a refund transaction.
-    Expired = 1,
+    Refundable = 4,
+
+    Completed = 5,
```

Comment:
Is there any need to migrate the swap_info table to update the swap status from Expired to Refundable/Completed else it will become WaitingConfirmation

Reply:
Sure, I am waiting with the migration until I'll finish the implementation.
My plan is to reset all of them to Initial which will include them in the next rescan by the swapper. It will update the right state for all of them.